lib/Chat/AutoComplete/SearchPlugin.php

```diff
@@ -65,6 +65,14 @@ public function setContext(array $context): void {
 	 * @since 13.0.0
 	 */
 	public function search($search, $limit, $offset, ISearchResult $searchResult) {
+		if ($this->room->getObjectType() === 'file') {
```

Comment:
btw jsut for the record:
I did it the other way around, so normal users are listed before users which are not in the room yet.

Reply:
Good point, I did not think on that. Then maybe it would be better to restore the previous code and explicitly remove duplicated elements?